lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -50,11 +53,7 @@ def to_representation(self, course_overview):
                 }
             },
             'course_image': course_overview.course_image_url,
-            'course_about': reverse(
-                'about_course',
-                kwargs={'course_id': course_id},
-                request=request,
-            ),
+            'course_about': None if not course_about_url else "http:{}".format(course_about_url),
```

Comment:
@verdanmahmood Yes, this is existing key and I am not sure if we can change this. It will crash already existing apps unless we create a new version for this API, which I would not recommend for a change with such a small scope.


@cahrens @nasthagiri Any comments why we omitted the `http/https` for cms? we do need it for lms and we have to re-append it back there.